uint8 IDLE = 1
uint8 INIT = 2
uint8 READY = 3
uint8 ACTIVE = 4
uint8 PAUSED = 5
uint8 STUCK = 6
uint8 SHUTDOWN = 7
uint8 EXTRACT = 8
int32 status
string text
